pkg/proxy/proxier.go

```diff
@@ -468,6 +468,10 @@ func (proxier *Proxier) OnUpdate(services []api.Service) {
 	glog.V(4).Infof("Received update notice: %+v", services)
 	activeServices := make(map[types.NamespacedName]bool) // use a map as a set
 	for _, service := range services {
+		// if PortalIP is "None", proxying is not required
+		if service.Spec.PortalIP == "None" {
```

Comment:
Probably we should define a helper function for determine whether PortalIP is set to an IP address.